spring-integration-core/src/main/resources/org/springframework/integration/config/spring-integration-5.0.xsd

```diff
@@ -5009,6 +5009,18 @@ See 'SmartLifecycleRoleController'.
 		</xsd:attribute>
 	</xsd:attributeGroup>
 
+	<xsd:attributeGroup name="maxFetchGroup">
+		<xsd:attribute name="max-fetch-size" type="xsd:string">
```

Comment:
Great! So, SIK is still must be fixed on the matter: https://github.com/spring-projects/spring-integration-kafka/blob/master/src/main/resources/org/springframework/integration/kafka/config/spring-integration-kafka-2.0.xsd#L82.

But yeah... Different story.
Will fix after cleaning the space on my PC :smile: